tensorflow/c/c_api.cc

```diff
@@ -581,6 +584,23 @@ TF_Buffer* TF_GetAllOpList() {
   return ret;
 }
 
+TF_Library* TF_LoadPluggableDeviceLibrary(const char* library_filename,
+                                          TF_Status* status) {
+  TF_Library* lib_handle = new TF_Library;
```

Comment:
Can you wrap the code here with `ifdefs` to disable this functionality on mobile. See TF_RegisterFilesystemPlugin for example in @yongtang's PR here: 
https://github.com/tensorflow/tensorflow/pull/43680/files

Right now we don't support runtime aspects of TF such as StreamExecutor on mobile.

Reply:
done.